### PR TITLE
Polish up documentation

### LINKED ISSUE #7: Polish readme files of all tools
The readme of all repositories and RS-DAT repo should be polished. So, this can be done by submitting PRs to relevant repos.

### PR DESCRIPTION
**Pull request by rogerkuou**
Fix #7

#### General
- [x] Landing page introduction of RS-DAT
- [x] Add content to citation tab (after #12)
- [x] Change relative links to absolute links

#### All component repos of RS-DAT
- [x] Update README

#### Deploy tab: 
- [ ] Bring long MD files to mkdocs [JDonSLURM user guide](https://github.com/RS-DAT/JupyterDaskOnSLURM/blob/main/user-guide.md)

#### Example Tab:
- [ ] Update [Phenology readme file](https://github.com/RS-DAT/JupyterDask-Examples/blob/main/03-phenology/README.md) with more explanation
- [ ] Include MobyLe example notebooks to: https://github.com/RS-DAT/JupyterDask-Examples

### CODE REVIEW

**Comment by rogerkuou** [comment]
Hi @fnattino, when you have time, can you help me review this PR?
I put all the unfinished tasks in https://github.com/RS-DAT/RS-DAT/issues/14.
The changes of relative links to abs links are not reflected in this repo. But when you compile with `mkdocs serve`, all the warnings regarding broken links should be gone.

File: docs/how_to_cite.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+
+## APA
+```
+Nattino, F., Grootes, M. W., Alidoost, F. (., Ku, O., & Chandramouli, P. RS-DAT [Computer software]. https://github.com/RS-DAT/RS-DAT
```
Comment (fnattino) [inline docs/how_to_cite.md:4]:
```suggestion
Nattino, F., Grootes, M. W., Alidoost, F., Ku, O., & Chandramouli, P. RS-DAT [Computer software]. https://github.com/RS-DAT/RS-DAT
```

File: docs/how_to_cite.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+```
+
+## BibTeX
+```
+@software{Nattino_RS-DAT,
+author = {Nattino, Francesco and Grootes, Meiert W. and Alidoost, Fakhereh (Sarah) and Ku, Ou and Chandramouli, Pranav},
```
Comment (fnattino) [inline docs/how_to_cite.md:10]:
```suggestion
author = {Nattino, Francesco and Grootes, Meiert W. and Alidoost, Fakhereh and Ku, Ou and Chandramouli, Pranav},
```

**Comment by fnattino** [comment]
Hi @rogerkuou, thanks, I have added few minor comments. I know I had already looked at the CITATION file, sorry for adding more suggested changes.

About the absolute links: thanks for adding them, still the links on the deployed website looks broken. I will open an issue to investigate this further, but feel free to merge on my side!

**Review by fnattino — COMMENTED** [review]
See comment above :)

**Comment by rogerkuou** [comment]
Thanks @fnattino. I applied the suggestions you made. Will merge this PR.

**Comment by rogerkuou** [comment]
> Hi @rogerkuou, thanks, I have added few minor comments. I know I had already looked at the CITATION file, sorry for adding more suggested changes.
> 
> About the absolute links: thanks for adding them, still the links on the deployed website looks broken. I will open an issue to investigate this further, but feel free to merge on my side!

After we merged this PR, the doc deploy action is triggered. Then the broken links work again. It seems that the ` include-markdown` plug-in requires this to get the fixed links in other repos.

**Pull request merged**